src/plugins/vis_types/xy/public/config/get_config.ts

```diff
@@ -91,15 +95,9 @@ export function getConfig(table: Datatable, params: VisParams): VisConfig {
  */
 const shouldEnableHistogramMode = (
   seriesParams: SeriesParam[],
-  yAspects: Aspect[],
   yAxes: Array<AxisConfig<ScaleContinuousType>>
 ): boolean => {
-  const bars = seriesParams.filter(({ type, data: { id: paramId } }) => {
-    return (
-      type === ChartType.Histogram && yAspects.find(({ aggId }) => aggId === paramId) !== undefined
-    );
-  });
-
+  const bars = seriesParams.filter(({ type }) => type === ChartType.Histogram);
```

Comment:
To have the same histogram enablement this needs to filter out `seriesParams` that are hidden, Otherwise this logic is pointless. I don't see a way to link a `seriesParam` to the corresponding `aspect` which is what tells you it is hidden. Do you know how we can determine what `seriesParams` are hidden in the **Data** tab?